build.gradle

```diff
@@ -27,5 +30,13 @@ dependencies {
     implementation 'org.slf4j:slf4j-api:1.7.30'
     testCompile group: 'junit', name: 'junit', version: '4.12'
     implementation 'com.upsolver:data-sources-interfaces:1.0.6'
+
+    compileOnly group: 'com.amazon.redshift', name: 'redshift-jdbc42', version: '1.2.1.1001'
```

Comment:
some of these are not used. Why add them?

Reply:
you are right. I think that we should discover other DBs too. These dependencies do not bother because they are added as `compileOnly`.